internal/commands/runcommand.go

```diff
@@ -6,17 +6,18 @@ import (
 	"time"
 
 	"github.com/UpCloudLtd/upcloud-cli/internal/config"
+	"github.com/UpCloudLtd/upcloud-cli/internal/log"
 	"github.com/UpCloudLtd/upcloud-cli/internal/output"
 	"github.com/UpCloudLtd/upcloud-cli/internal/resolver"
 	internal "github.com/UpCloudLtd/upcloud-cli/internal/service"
 	"github.com/spf13/cobra"
 )
 
 func commandRunE(command Command, config *config.Config) func(cmd *cobra.Command, args []string) error {
-
 	switch typedCommand := command.(type) {
 	case NoArgumentCommand:
 		return func(cmd *cobra.Command, args []string) error {
+			log.Debug("runcommand.go", "commandRunE() NoArgumentCommand", "Cmd path:", cmd.CommandPath(), "Arguments:", args)
```

Comment:
not sure if the 'scope' here should be the filename? 

We can get the filename + line automatically with [`runtime.Caller()`](https://golang.org/pkg/runtime/#Caller) so if we want to use it as the scope, would suggest adding some sort of an automatic way in order to reduce possible developer errors (for example, moving some code around).

Reply:
This is not very idiomatic flume usage. One should create a flume instance per package or per logical part of the application. That way one can control the logging level per each logger instance.

Reply:
Moreover, flume already supports adding caller information with a relevant option so logging the filename is redundant.

Reply:
So hmm... How should we init the logger in a way that it should be easy to use for any component? I am starting to think my tryouts here are pretty useless 😅 which is ok, still learning to use go and so on. Since the AddCaller wouldn't work here, since the only path it would show is the log.go pckg

Did think about having the pckg:s create a flume instance with configs. Then the debugMode would be set in the log.go pckg to be called when creating the logger instance. Though unsure if that is good or bad way of doing it and seems a bit silly to even use a different pckg just for debugMode.

So any ideas are welcome :)

Reply:
Changed the thought process. And each pckg creates the flume logger and then passes it to the log.go pckg. Though still the caller is in form `@:orig-pckg@log/log.go:39`

Reply:
You can add skipcaller option to skip the wrapping pkg.